docs/providers/provider-protocol-reference.md

```diff
@@ -4,183 +4,195 @@ category: 63e0c7cf7166bf006d3b4117
 slug: provider-protocol-reference
 ---
 
-All functions described here are bindings generated with fp-bindgen.  The key/values to use for these payloads are exactly the ones you can find in `fiberplane::models` [crate documentation](https://docs.rs/fiberplane-models/latest/fiberplane-models)
+All functions described here are bindings that are generated from our [provider protocol](https://github.com/fiberplane/fiberplane/tree/main/fiberplane-provider-protocol) and which are publish as our [provider bindings crate](https://docs.rs/fiberplane-provider-bindings/latest/fiberplane_provider_bindings/). Please refer to the crate for more details on any of the types mentioned below.
 
-## Imported functions
+We use [`fp-bindgen`](https://github.com/fiberplane/fp-bindgen) for generating the bindings themselves.
 
-These functions are usable as long as you import `fiberplane_pdk::prelude::*` in your library code. These functions exist to allow you to use OS-level features by asking the runtime for information.
+## Exported functions
 
-For the Fiberplane Daemon, these functions are implemented in [Provider Runtime crate](https://github.com/fiberplane/fiberplane-rs/blob/main/fiberplane-provider-protocol/fiberplane-provider-runtime/src/spec/mod.rs), should you want to inspect the source.
+Exported functions are functions that your provider implements, and that the runtime will call. In order to make those exported functions usable in Fiberplane, they must be annotated with `#[pdk_export]`.
 
-### `make_http_request`
+### `get_supported_query_types`
+
+It is advised to use the [Plugin Development Kit](https://docs.rs/fiberplane-pdk/latest/fiberplane_pdk/) rather than implementing this function manually. The [`pdk_query_types!`](https://docs.rs/fiberplane-pdk-macros/latest/fiberplane_pdk_macros/macro.pdk_query_types.html) macro can be used to generate this function for you. See the [tutorial](doc:create-a-provider) for more information about using the PDK.
```

Comment:
```suggestion
It is advised to use the [Provider Development Kit](https://docs.rs/fiberplane-pdk/latest/fiberplane_pdk/) rather than implementing this function manually. The [`pdk_query_types!`](https://docs.rs/fiberplane-pdk-macros/latest/fiberplane_pdk_macros/macro.pdk_query_types.html) macro can be used to generate this function for you. See the [tutorial](doc:create-a-provider) for more information about using the PDK.
```